src/lib/stark/ledger/createSignedTransfer.js

```diff
@@ -28,20 +29,20 @@ module.exports = async (
   await dvf.token.provideContractData(eth, token, tokenAddress, transferQuantization)
 
   const starkSignature = await eth.starkSignTransfer_v2(
-      starkPath,
-      tokenAddress,
-      token === 'ETH' ? 'eth' : 'erc20',
-      transferQuantization,
-      null,
-      starkPublicKey.x,
-      sourceVault,
-      destinationVault,
-      amountTransfer,
-      nonce,
-      expireTime,
-      null,
-      null
-    )
+    starkPath,
+    tokenAddress,
+    token === 'ETH' ? 'eth' : 'erc20',
+    transferQuantization,
+    null,
+    receiverPublicKey || starkPublicKey.x,
```

Comment:
More for the future if this needs to be merged quickly, but we could probably enforce callers of this method to input `receiverPublicKey` explicitly and avoid the `||` (but would involve modifying existing methods)